python/suricata/ctl/main.py

```diff
@@ -1,4 +1,5 @@
-# Copyright (C) 2018 Open Information Security Foundation
+#!/usr/bin/env python
+# Copyright (C) 2018-2023 Open Information Security Foundation
```

Comment:
As this file is never intended to be directly executed, I wonder if we should have the shebang here..

Reply:
You're right. It doesn't make sense. Should remove it..